src/parameterizations/lateral/MOM_lateral_mixing_coeffs.F90

```diff
@@ -643,20 +652,30 @@ subroutine calc_slope_functions(h, tv, dt, G, GV, US, CS, OBC)
       call calc_isoneutral_slopes(G, GV, US, h, e, tv, dt*CS%kappa_smooth, CS%use_stanley_iso, &
                                   CS%slope_x, CS%slope_y, N2_u=N2_u, N2_v=N2_v, halo=1, OBC=OBC)
       call calc_Visbeck_coeffs_old(h, CS%slope_x, CS%slope_y, N2_u, N2_v, G, GV, US, CS)
+    elseif (CS%use_gradient_model) then
+      call calc_slope_functions_with_gradient_model(h, G, GV, US, CS, e, uh, vh)
     else
       call calc_slope_functions_using_just_e(h, G, GV, US, CS, e)
     endif
   endif
 
+!!  if (CS%use_gradient_model) then
```

Comment:
Please remove this extra commented out call to `calc_slope_functions_with_gradient_model()`.